package.json

```diff
@@ -36,19 +37,26 @@
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
+    "jest": "^26.6.3",
+    "nodemon": "^2.0.7",
     "pkg": "^4.4.2",
     "prettier": "^2.0.5",
+    "rimraf": "^2.6.3",
     "source-map-support": "^0.5.16",
-    "tsc-watch": "^1.1.39",
+    "ts-jest": "^26.5.5",
+    "ts-node": "^9.1.1",
     "tslint": "^5.20.1",
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^8.0.1",
     "typescript": "^3.7.4"
   },
   "scripts": {
-    "start": "tsc-watch --onSuccess \"node -r source-map-support/register ./build/app.js\"",
-    "build": "tsc",
-    "deploy": "pkg . -t node10-alpine-x64,node10-linux-x64,node10-macos-x64 --out-path bin/"
+    "build": "rimraf build && tsc",
+    "coverage": "jest --coverage",
+    "dev": "docker compose up -d && nodemon --exec ts-node ./src/index.ts --watch src",
+    "deploy": "rimraf bin && pkg . -t node10-alpine-x64,node10-linux-x64,node10-macos-x64 --out-path bin/",
+    "start": "npm run build && node build/index.js",
+    "test": "jest --watch"
```

Comment:
In here I changed `app.ts` to `index.ts`. `npm start` should behave the same as it did before for, and should work great for non-docker testing. I added `dev` for local docker-based development